src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs

```diff
@@ -2,73 +2,36 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
 using System.Collections.Generic;
-using System.Collections.Immutable;
 using System.Linq;
 using System.Reflection;
 using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
 using Microsoft.CodeAnalysis.Shared.Utilities;
 
 namespace Microsoft.CodeAnalysis.CodeFixes
 {
-    internal abstract class FixSomeCodeAction : CodeAction
+    internal abstract class FixSomeCodeAction : FixAllCodeAction
```

Comment:
1. can we doc.  just seeing hte name/inheritance is confusing. e.g. how is 'Fixing some... fixing all'? :)  
2. can you name this AbstractFixSomeCodeAction?

Reply:
I have added doc comment and renamed this to `AbstractFixAllCodeFixCodeAction`, which is the real purpose of this type: it is an abstract fix all code action for code actions registered by code fix providers, with two concrete sub-types.